python-requirements.txt

```diff
@@ -6,12 +6,15 @@
 hjson
 libcst
 mako
+pluralizer
```

Comment:
These will require a airgapped custom image update just an FYI. I can perform the update then merge this. (I'm in the process of automating it, so this will be a good test case :) ).

Reply:
SG, thanks for letting me know! I thought the new dependencies wouldn't be an issue since this test is tagged as `manual` but I'm not sure referencing those in a `BUILD` file could cause issues.

Reply:
So we have a [repo rule](https://cs.opensource.google/opentitan/opentitan/+/master:third_party/python/pip.bzl;drc=e66ee7f48dd5525c2a657a4671e1068af6c24a13;l=95) that ingests the `python-requirements.txt` file and preps the python wheels cache (to enable offline builds). This gets run when the workspace is loaded, regardless of what BUILD rule is invoked.

Reply:
I see, thanks for the explanation!